suite-native/app/ios/Podfile.lock

```diff
@@ -871,4 +871,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: d37876aaf00e7f8309b694188119b139b8452ca1
 
-COCOAPODS: 1.13.0
+COCOAPODS: 1.12.1
```

Comment:
Cocoapods is a local dependency so don't worry about it in PR's please. I updated mine so now it's fine but if you want to worry about it in PR's we should enforce version of cocoapods to be used which can be done
https://github.com/trezor/trezor-suite/pull/9694/commits/0347bcf7e7df5749119a6bd8389afa63fe5c71ec

Reply:
IMO this file should be never pushed at all if the commit does not introduce a new installed pod. Otherwise it is unrelated change.  Thanks for resolving this.